ipaserver/plugins/automount.py

```diff
@@ -610,8 +611,11 @@ def check_key_uniqueness(self, location, map, **keykw):
         entries = self.methods.find(location, map, automountkey=key)['result']
         if len(entries) > 0:
             if key == DIRECT_MAP_KEY:
+                if 'quiet' in keykw:
+                    newkw = deepcopy(keykw)
```

Comment:
Do you need a deepcopy here? deepcopy recurses into values and makes copies of nested values. I think ``keykw.copy()`` should be good enough